test/config/jobconfig.json

```diff
@@ -1,52 +1,53 @@
 [
   {
     "jobType": "archive",
-    "create": [
-      {
-        "action": {
-          "actionType": "log"
-        }
-      }
-    ],
-    "update": {
-      "action": {
+    "create": {
+      "actions": {
         "actionType": "log"
       }
     },
-    "post": [
-    {
-      "action": {
-        "actionType": "email",
-        "mailer": {
-          "host": "smtp.myinstitution.ext",
-          "port": 465,
-          "secure": true,
-          "auth": {
-            "user": "scicatuser",
-            "pass": ""
-          }
-        },
-        "to": "{{ join job.recipients ' '}}",
-        "from": "scicat@example.com",
-        "subject": "Archive job {{job.pid}} has completed.",
-        "body": "Dear {{job.owner}},\n\nArchive job {{job.pid}} has completed. Results: {{job.status}}\n"
+    "update": {
+      "auth": "#all",
```

Comment:
Here's an example of configuring authorization at the JobOperation level.

Reply:
We should use this opportunity to provide a plain description of the job and workflow, and create the matching configuration.
This would be a good exercise for us to find strengths and shortcomings and to be able to create good quality documentation